HIP/hip-583.md

```diff
@@ -2,221 +2,266 @@
 hip: 583
 title: Expand alias support in CryptoCreate & CryptoTransfer Transactions
 author: Nana Essilfie-Conduah <@Nana-EC>
-working-group: Danno Ferrin <@shemnon>, Atul Mahamuni <@atul-hedera>, Richard Bair <@rbair23>, Ali Katamjani <ali@swirldslabs.com>, Michael Tinker <@tinker-michaelj>, Mohsin Qamar <@mohsin-hedera>, Leemon Baird <@lbaird>
+working-group: Danno Ferrin <@shemnon>, Atul Mahamuni <@atul-hedera>, Richard Bair <@rbair23>,
+               Ali Katamjani <ali@swirldslabs.com>, Michael Tinker <@tinker-michaelj>, Mohsin Qamar <@mohsin-hedera>,
+               Leemon Baird <@lbaird>
 type: Standards Track
 category: Service
 needs-council-approval: Yes
 status: Accepted
 last-call-date-time: 2022-10-17T07:00:00Z
 created: 2022-09-20
 discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/593
-updated: 2022-09-29, 2022-10-03, 2022-10-20, 2022-11-11
+updated: 2022-09-29, 2022-10-03, 2022-10-20, 2022-11-11, 2023-02-07
 requires: 32
```

Comment:
Should this supersede 631 or how do we go about indicating 631 will not be implemented?

Reply:
631 depends on this one, not the other way around. I think that means this one doesn't supersede the other, but that the other is out of date and needs to be updated.